client/src/components/screens/PatientCreation/components/FormModal/index.tsx

```diff
@@ -10,13 +10,12 @@ import { FormItemProps } from 'antd/lib/form';
 import { useForm } from 'antd/lib/form/Form';
 import moment from 'moment';
 import { RadioChangeEvent } from 'antd/lib/radio';
-import { set } from 'lodash';
+import { get, set } from 'lodash';
```

Comment:
using lodash that way, it make the bundle size explode. In CQDG and KidsFirst this pattern is proscribe.

Instead use
```javascript
import get from 'lodash/get';
import set from 'lodash/set';
```

A little bit verbose, but if use over the app you'll save around 100k.

Reply:
Ok, I'll create a new PR with the changes